lib/cleaned_output.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+require 'colorize'
+
+class CleanedOutput
+  def initialize(output:, morph_api_key: ENV['MORPH_API_KEY'])
```

Comment:
it feels a little odd to me to have this class so tightly coupled to Morph-specific concepts like this, or for a class like this to be peering into ENV. This seems like it would be a lot more extensible if the argument here was something like `redact: *strings`, for a list of things you want redacted in the output.